pyndl/ndl_parallel.pyx

```diff
@@ -26,6 +27,12 @@ test_index *= test_outcome_index
 test_index += test_cue_index
 assert test_index == 18446744069414584320
 
+cdef enum ErrorCode:
+    NO_ERROR = 0
+    MAGIC_NUMBER_DOES_NOT_MATCH = 1
+    VERSION_NUMBER_DOES_NOT_MATCH = 2
+    INITIAL_ERROR_CODE = 3
```

Comment:
Would name this `undefined error` because it occurs only if there's not alright (error=0), but did not enter a defined error state.